lib/fx/adapters/postgres/functions.rb

```diff
@@ -53,6 +61,25 @@ def functions_from_postgres
         def to_fx_function(result)
           Fx::Function.new(result)
         end
+
+        def schema_aware_name(function)
+          if function.fetch("schema") == function.fetch("current_schema")
+            function.fetch("name")
+          else
+            "#{function.fetch("schema")}.#{function.fetch("name")}"
+          end
+        end
+
+        def schema_aware_definition(function)
```

Comment:
The reason this method is necessary is because PG itself is not schema-aware when returning the function definition, and will, for example, include the fully-qualified name (`public.something`) even if `current_schema()` matches the function's schema.

This is probably the correct behavior on PG's part, but it means that for `schema.rb` purposes, we should strip it. (See [here](https://github.com/teoljungberg/fx/pull/140/files#r1459966705) for the reasoning -- this how Rails handles enum names in a way that enables a single `schema.rb` to be valid & shareable in many contexts that may not be configured to have the same default PG schema.)